### DISCUSSION TITLE
Wrong Phong Lighting Effect

### DISCUSSION THREAD

**Discussion by pskpg** [pelicanmapping/osgearth#1977] [Support Q&A] [answered]
Hi Glenn,
I'm using oe3.2. And I got a wrong phong lighting effect when running "osgearth_lights simple.earth". Here are the screenshots:
![1](https://user-images.githubusercontent.com/20101178/153756852-9a9770d7-92d4-4a4a-b7fa-bb33c17cedb8.JPG)
![2](https://user-images.githubusercontent.com/20101178/153756858-7ae18716-5058-4c88-be26-956cc8b35810.jpg)
![3](https://user-images.githubusercontent.com/20101178/153756957-ceeee305-5d68-41f5-813e-7af2897edea3.JPG)
![4](https://user-images.githubusercontent.com/20101178/153756961-ae5a0410-3e11-4fce-9d6d-6fa98c4d3bfa.JPG)
![5](https://user-images.githubusercontent.com/20101178/153756970-d757c466-39b3-4752-b050-1aba5eea3d7b.JPG)

How to fix it?

**Comment by RhythmOfTension** [comment, marked as answer]
Looks like https://github.com/gwaldron/osgearth/issues/1699

**Reply by pskpg** [reply]
Thank you so much! 
I'll rebuild osg and try later!

**Reply by pskpg** [reply]
Hi, RhythmOfTension,
I've rebuild OSG with OPENGL_PROFILE=GL3 and OSG_GL3_AVAILABLE on, and the light on earth is nomal. But there comes another bug: osgviewer can load but cannot show any model. When I run "osgviewer cow.osg", I got a totally blue screen. Have you met this problem?
